### ISSUE TITLE
When comparing files, the files do not use the same setting for Word Wrapping

### ISSUE THREAD

**Issue by RomanPiperMVTec** [eclipse-theia/theia#14620]
### Bug Description:
When comparing files, the files do not use the same setting for Word Wrapping. Therefore the same line is not displayed in the same way when comparing.

### Steps to Reproduce:

1. Activate Word Wrap (>View: Toggle Word Wrap)
2. make a copy of a file with long lines (e.g. a long string)
3. Compare files (e.g. >Compare: Select For Compare, >Compare: Compare with Selected )

![Screenshot 2024-12-13 095001](https://github.com/user-attachments/assets/9482dd46-3bef-4d80-ac2a-f05d142895b9)

### Additional Information

- Operating System: Windows 11 Pro
- Theia Version: 1.55.0

**Comment by tsmaeder** [comment]
I don't think it even makes sense to use "word wrap" in the diff editor: the editor falsely indicates that there is a diff in the versions when the diff is actually in the display.

**Comment by hannescmarits** [comment]
@tsmaeder not supporting "word wrap" may be correct, but still it should the same mode should be used for both views.
